tests/conftest.py

```diff
@@ -1,138 +1,532 @@
-"""Pytest configuration for the aiida-firecrest tests.
-
-This sets up the Firecrest server to use, and telemetry for API requests.
-"""
 from __future__ import annotations
 
-from functools import partial
-from json import load as json_load
+from dataclasses import dataclass
+import hashlib
+import json
+import os
 from pathlib import Path
+import random
+import shutil
+import stat
 from typing import Any, Callable
 from urllib.parse import urlparse
 
-from _pytest.terminal import TerminalReporter
-import firecrest as f7t
+from aiida import orm
+import firecrest
+import firecrest.path
 import pytest
 import requests
-import yaml
 
-from aiida_firecrest.utils_test import FirecrestConfig, FirecrestMockServer
 
+class Values:
+    _DEFAULT_PAGE_SIZE: int = 25
```

Comment:
Is there a reason why you don't just use `FirecrestScheduler._DEFAULT_PAGE_SIZE`?